src/registrar/admin.py

```diff
@@ -222,6 +208,20 @@ def group(self, obj):
         "date_joined",
     ]
 
+    list_filter = (
+        "is_active",
+        "groups",
+    )
+
+    # Let's define First group
+    # (which should in theory be the ONLY group)
+    def group(self, obj):
+        if obj.groups.filter(name="full_access_group").exists():
+            return "Full access"
+        elif obj.groups.filter(name="cisa_analysts_group").exists():
+            return "Analyst"
+        return ""
+
```

Comment:
This was a merge conflict. I just moved it down so it was grouped with the other functions, but I can move it back if that is how it was in the original code

Reply:
Ideally, keep this down and add a small comment around line 141 indicating that `group` in `list_display` is custom defined?

Reply:
Ah so like separating it back into its own location. Yeah, makes sense to me

Reply:
It can stay where it is, but we should note that group is a custom column/prop defined in admin.py as opposed to the model.